### PR TITLE
Fix all modules and usage examples to compile without error on 2.23.80

### PR DESCRIPTION
**Pull request by markk**

### CODE REVIEW

File: layout/display/control-points.ily [hunk 1]
```diff
@@ -1,4 +1,4 @@
-\version "2.19.22"
+\version "2.23.80"
```
Comment (jeanas) [inline layout/display/control-points.ily:1]: If OLL requires 2.23+, this file can be removed entirely due to the addition of the ` \vshape` command.

File: rests/auto-merge/module.ily [hunk 2]
```diff
@@ -1,4 +1,4 @@
-\version "2.18.0"
```
Comment (jeanas) [inline rests/auto-merge/module.ily:1]: Does this snippet do anything differently from LilyPond's built-in `Merge_rests_engraver` and from `Merge_mmrests_engraver` that was added in 2.21.6?

File: layout/display/control-points.ily (hunk 1 above)
Comment (markk) [reply layout/display/control-points.ily:1]: Yes. I was going to do a cull of redundant/duplicated code as a second pass. Will put this one on that list.

File: rests/auto-merge/module.ily (hunk 2 above)
Comment (markk) [reply rests/auto-merge/module.ily:1]: Ditto

**Comment by jeanas** [comment]
Let's merge.

**Pull request merged**